mainlyetcetera/rotten_tomatillos
# Feature/return to main view

## Is this a fix or a feature?
- feature
## What is the change/fix?
- add link on details view to return to main view
- ensure route renders are rendering both views properly
## Where should the reviewer start?
- `src/components/Details/Details.js`
## How should this be tested?
- pull down branch (not to main)
- run `npm run start`
- navigate to `http://localhost:3000`
- click on a movie (anywhere in the card should be fine)
- click on "Back to Movies" link
- observe that you returned to the main view
## This PR is related to which issues:
- this closes #18
